### DISCUSSION TITLE
How do I cite quarto in an academic paper?

### DISCUSSION THREAD

**Discussion by jchiquet** [quarto-dev/quarto-cli#4923] [Q&A] [answered]
At a conference, we present our [new academic journal](https://computo.sfds.asso.fr/), which offers quarto-based templates for submissions. We would like to show quarto among the references: is there a canonical BibTeX entry discussed/available somewhere?

Thanks

**Comment by cscheid** [comment, marked as answer]
(As a former academic: thanks for doing that!)

The citation info is on https://github.com/quarto-dev/quarto-cli/ (this is supported by many repos, not just us!)

<img width="981" alt="image" src="https://user-images.githubusercontent.com/285675/226991553-83769d2e-cfdf-4037-9ebc-167db00a8295.png">

**Reply by jchiquet** [reply]
Great ! Thank you for your quick response. I was not yet aware of this feature.
